src/clientmetricreport/ClientMetricReport.ts

```diff
@@ -15,4 +17,9 @@ export default interface ClientMetricReport {
    * Gets video client media metrics
    */
   getObservableVideoMetrics?(): { [id: string]: { [id: string]: {} } };
+
+  /**
+   * Gets raw RTCStatsReport from `RTCPeerConnection.getStats()` API
+   */
+  getRawMetricReport?(): RawMetricReport;
```

Comment:
Should we remove or deprecate https://github.com/aws/amazon-chime-sdk-js/blob/main/src/audiovideocontroller/DefaultAudioVideoController.ts#L219 as they do the same thing?

Reply:
I prefer not to change it.
* it offers the most flexible way to handle the metrics, give all control to the builder
* in the design meeting, we didn't decide to remove this

It's not necessary to remove it so keeping it can lower the bar for V2 -> V3 migration for builders.

Reply:
i think we can at least mark it as deprecated and recommend using the callback. Its a tiny bit inefficient to re-call it on different intervals. Maybe we could provide control over call interval.

Reply:
Added deprecation comment.